src/test/java/seedu/address/model/card/FillBlanksCardTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.card;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_FILLBLANK_BACK_SET;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_FILLBLANK_FRONT_SET;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import org.junit.Test;
+
+import seedu.address.testutil.Assert;
+
+//@author shawnclq
+public class FillBlanksCardTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        Assert.assertThrows(NullPointerException.class, () -> new FillBlanksCard(null,
+                VALID_FILLBLANK_FRONT_SET, VALID_FILLBLANK_BACK_SET));
+        Assert.assertThrows(NullPointerException.class, () -> new FillBlanksCard(UUID.randomUUID(),
+                null, VALID_FILLBLANK_BACK_SET));
+        Assert.assertThrows(NullPointerException.class, () -> new FillBlanksCard(UUID.randomUUID(),
+                VALID_FILLBLANK_FRONT_SET, null));
+        Assert.assertThrows(NullPointerException.class, () ->
+                new FillBlanksCard(null, VALID_FILLBLANK_BACK_SET));
+        Assert.assertThrows(NullPointerException.class, () ->
+                new FillBlanksCard(VALID_FILLBLANK_FRONT_SET, null));
+    }
+
+    @Test
+    public void constructor_invalidParam_throwsIllegalArgumentException() {
+        String invalidParam = " ";
+        List<String> invalidFrontSet = Arrays.asList(new String[] {invalidParam, "Hello", "World"});
+        List<String> invalidBackSet = Arrays.asList(new String[] {invalidParam, "Hello"});
+        Assert.assertThrows(IllegalArgumentException.class, () -> new FillBlanksCard(UUID.randomUUID(),
```

Comment:
Create a constructor for FillBlanksCard that accepts fewer arguments and instantiates a UUID by default.